src/pages/Contact.tsx

```
@@ -0,0 +1,161 @@
+    image: cImg4,
+    address: 'Rotonda Loredana 5, Sesto Ingrid, Messina, Italy',
+    phone: '000 - 123 - 456 - 789',
+    email: 'support@example.com',
+    hours: 'Mon - Fri : 8am - 5pm',
+  },
+]
+
+export default function Contact() {
+  useEffect(() => {
+    // Prevent Leaflet icon errors
+    delete (L.Icon.Default.prototype as any)._getIconUrl
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)